### PR TITLE
Feature/textfield test

### PR DESCRIPTION
**Pull request by lautimfernandez**
## Motivation
Adding tests and some fixes discovered with tests.

## Affected component
Andes Textfield.

## Checks
Are you sure you followed all those steps? In such case, let's say with me "I solemnly declare that ...":
- [ ] I have coded an example inside the Demo App,
- [X] I have added proper tests,
- [ ] I have modified the [Changelog](https://github.com/mercadolibre/fury_andesui-android/blob/develop/CHANGELOG.md) file,
- [ ] I have updated GitHub wiki,
- [ ] I have the explicit approval of one or more members of the UX Team,
- [ ] I have updated the version in gradle.properties file.

## Change type
This is easy, let us know what this code is about:
- [ ] This code adds a new component
- [X] This code includes improvements to an existent component
- [ ] This code improves or modifies ONLY the Demo App.

## Check common errors
Whether you are an Android or iOS developer and if you're still there then we'll give you a present:
https://proandroiddev.com/how-to-maximize-androids-ui-reusability-5-common-mistakes-cb2571216a9f

### CODE REVIEW

**Comment by furyGorgoryAdmin** [comment]
### Informative Menu <h2>  ### 

### Use any of these tags in a pull request description or leave a comment ### 
| Tag | Description |
|---|---|
| **[canary]**| Creates a build that can only be deployed with canary strategy. | 
| **[build]** |  Creates version or build from a branch that has been tagged as "no releasable". | 
| **[name:version-name]** |Specify the name of the build. For example: [build] [name:1.2.3-name] or [canary] [name:1.2.3-name].|
| **[major]** | To automatically increase the first position of the version. For example: [build] [major] or [canary] [major].|
| **[minor]** | To automatically increase the second position of the version. For example: [build] [minor] or [canary] [minor]. |
| **[patch]** | To automatically increase the third position of the version. For example: [build] [patch] or [canary] [patch].|  

[For more information about ci-cd](http://docs.furycloud.io/?lang=esp&section=businessassurance)
[For more information about semver](https://semver.org/#summary)

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt [hunk 1]
```diff
@@ -458,7 +458,7 @@ class AndesTextfield : ConstraintLayout {
      */
     fun setSuffix(text: String) {
         rightContent = AndesTextfieldRightContent.SUFFIX
-        val suffix: TextView = leftComponent.getChildAt(0) as TextView
+        val suffix: TextView = rightComponent.getChildAt(0) as TextView
```
Comment (gtchantouria) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt:461]: ja como no lo vimos y cómo no me saltó en las pruebas de la testapp esto no??!!

Comment (lautimfernandez) [reply components/src/main/java/com/mercadolibre/android/andesui/textfield/AndesTextfield.kt:461]: unos colgados jajaj

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt [hunk 2]
```diff
@@ -69,6 +69,7 @@ internal object AndesIconTextfieldContent : AndesTextfieldContentInterface() {
                 OfflineIconProvider(context).loadIcon("andes_ui_placeholder_imagen_24") as BitmapDrawable,
                 context,
                 R.color.andes_gray_800.toAndesColor()))
+        icon.contentDescription = "icon"
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt:72]: Podemos usar constantes acá?

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt [hunk 3]
```diff
@@ -95,6 +96,7 @@ internal object AndesValidatedTextfieldContent : AndesTextfieldContentInterface(
                 context,
                 R.color.andes_green_500.toAndesColor())
         )
+        validated.contentDescription = "validated icon"
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt:99]: Same

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt [hunk 4]
```diff
@@ -105,13 +107,15 @@ internal object AndesClearTextfieldContent : AndesTextfieldContentInterface() {
...
-        val validated =  SimpleDraweeView(context)
-        validated.setImageDrawable(buildColoredBitmapDrawable(
+        val clear = SimpleDraweeView(context)
+        clear.setImageDrawable(buildColoredBitmapDrawable(
                 OfflineIconProvider(context).loadIcon("andes_ui_close_24") as BitmapDrawable,
                 context,
                 R.color.andes_gray_450.toAndesColor())
         )
-        return validated    }
+        clear.contentDescription = "clear"
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt:116]: Same

File: components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt [hunk 5]
```diff
@@ -105,13 +107,15 @@ internal object AndesClearTextfieldContent : AndesTextfieldContentInterface() {
     override fun rightMargin(context: Context): Int = context.resources.getDimension(R.dimen.andes_textfield_clear_right_margin).toInt()
 
     override fun component(context: Context): SimpleDraweeView {
-        val validated =  SimpleDraweeView(context)
-        validated.setImageDrawable(buildColoredBitmapDrawable(
+        val clear = SimpleDraweeView(context)
+        clear.setImageDrawable(buildColoredBitmapDrawable(
                 OfflineIconProvider(context).loadIcon("andes_ui_close_24") as BitmapDrawable,
```
Comment (jiurchuk) [inline components/src/main/java/com/mercadolibre/android/andesui/textfield/content/AndesTextfieldContentInterface.kt:112]: Same

File: components/src/test/java/com/mercadolibre/android/andesui/textfield/AndesTexfieldContentInterfaceTest.kt [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+import org.robolectric.RuntimeEnvironment
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(constants = BuildConfig::class, sdk = [Build.VERSION_CODES.LOLLIPOP])
+class AndesTextfieldContentInterfaceTest {
+    private var context = RuntimeEnvironment.application
+    private lateinit var contentInterface: AndesTextfieldContentInterface
+
+
```
Comment (jiurchuk) [inline components/src/test/java/com/mercadolibre/android/andesui/textfield/AndesTexfieldContentInterfaceTest.kt:39]: Empty lines

**Pull request merged**